### ISSUE TITLE
Incorrect `dcos node` formatting

### ISSUE THREAD

**Issue by jsancio** [dcos/dcos-cli#677]
Please answer the following questions before submitting your issue. Thanks!

### What version of DC/OS + DC/OS CLI are you using (`dcos --version`)?
If DC/OS CLI version < 0.4.5, DC/OS version can be found at `http://dcos_url/dcos-metada/dcos-version.json`
```
$ dcos --version
dcoscli.version=0.4.8
dcos.version=N/A
dcos.commit=N/A
dcos.bootstrap-id=N/A
```

### What operating system and version are you using?
Ubuntu 16.04

### What did you do?
If possible please provide a recipe for reproducing.
`$ dcos node --help`

### What did you expect to see?
```
Usage:
    dcos node --help
    dcos node --info
    dcos node [--json]
    dcos node log [--follow --lines=N --leader --master --mesos-id=<mesos-id> --slave=<slave-id>]
    dcos node ssh [--option SSHOPT=VAL ...]
                  [--config-file=<path>]
                  [--user=<user>]
                  [--master-proxy]
                  (--leader | --master | --mesos-id=<mesos-id> | --slave=<slave-id>)
                  [<command>]
    dcos node diagnostics create (<nodes>)...
    dcos node diagnostics delete <bundle>
    dcos node diagnostics download <bundle> [--location=<location>]
    dcos node diagnostics (--list | --status | --cancel)
                          [--json]
```

### What did you see instead?
Incorrect formatting:
```
Usage:
    dcos node --help
    dcos node --info
    dcos node [--json]
    dcos node log [--follow --lines=N --leader --master --mesos-id=<mesos-id> --slave=<slave-id>]
    dcos node ssh [--option SSHOPT=VAL ...]
                  [--config-file=<path>]
                  [--user=<user>]
                  [--master-proxy]
                  (--leader | --master | --mesos-id=<mesos-id> | --slave=<slave-id>)
                  [<command>]
    dcos node diagnostics create (<nodes>)...
    dcos node diagnostics delete <bundle>
    dcos node diagnostics download <bundle> [--location=<location>]
    dcos node diagnostics (--list | --status | --cancel)
                       [--json]
```

**Comment by tamarrow** [comment]
#693

**Comment by tamarrow** [comment]
fixed by #693
